docs/reference-guide/exposing-system-metrics.md

```diff
@@ -0,0 +1,257 @@
+---
+id: exposing-system-metrics
+title: Exposing system metrics (Enterprise)
+sidebar_label: Exposing system metrics
+---
+
+In production systems, monitoring of applications is crucial, and that includes databases as well. Since Memgraph v2.8,
+an HTTP server has been added in order to expose system metrics, giving developers ability to track information about transactions,
+query latencies, snapshot recovery latencies, triggers, bolt messages, indexes, streams, and many more.
```

Comment:
```suggestion
In production systems, monitoring of applications is crucial, and that includes databases as well. Memgraph allows tracking information about transactions, query latencies, snapshot recovery latencies, triggers, bolt messages, indexes, streams, and many more.
```

the docs are versioned so this will appear only in version 2.8 indicating that the feature became available then